server.go

```diff
@@ -64,8 +64,10 @@ type Server struct {
 	fs       fs.FS
 	template *template.Template // TODO: use preparsed templates
 
-	DisallowUnknownFields bool // If true, the server will return an error if the request body contains unknown fields. Useful for quick debugging in development.
-	DisableOpenapi        bool // If true, the routes within the server will not generate an OpenAPI spec.
+	acceptedContentTypes []string
```

Comment:
Why is it added back on the Server struct?

Reply:
Ah I had a conflict on a rebase.